.github/workflows/mirror-php-package.yml

```
@@ -0,0 +1,39 @@
+            - name: Checkout monorepo
+              uses: actions/checkout@v4
+
+            - name: Setup WooCommerce Monorepo
+              uses: ./.github/actions/setup-woocommerce-monorepo
+            
+            - name: Build package for mirroring
+              run: |
+                  pnpm --filter "${{ github.event.inputs.package }}" build:composer-package
+
+            - name: Grab package build directory
+              run: echo "mirror_dir=$( pnpm --filter ${{ github.event.inputs.package }} list --json | jq --compact-output '.[]' | jq --raw-output '( .path + "/build")' )" >> $GITHUB_ENV
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2086:info:1:156: Double quote to prevent globbing and word splitting

(shellcheck)